### ISSUE TITLE
DO NOT DOWNLOAD THIS!! IT WILL STEAL YOUR ACCOUNT

### ISSUE THREAD

**Issue by TheSatanicAngeI** [fahrennheit/zombsroyaleio-hack#19]
"You guys know very well that this cheat is most likely a keylogger or a tracker because it never worked nor has the components for a functional aimbot right? Please tell me you guys are smart enough to not download this..."

**Comment by IceHacks** [comment]
Yes, my friends told me to fix this some time ago. I said "You idiots. It has a NetFlix keylogger. It never worked."

**Comment by TheSatanicAngeI** [comment]
@IceHacks LMAO
